src/main.cpp

```diff
@@ -344,26 +350,32 @@ void getCommandSchemaJson(JsonVariant json)
     controller["type"] = "integer";
     controller["minimum"] = 1;
     controller["maximum"] = PWM_CONTROLLER_COUNT;
+    controller["description"] = "Specify which PCA controlelr the strip is attached";
   }
 
   JsonObject strip = channelProperties.createNestedObject("strip");
   strip["type"] = "integer";
   strip["minimum"] = 1;
   strip["maximum"] = PWM_CHANNEL_COUNT;
+  strip["description"] = "What strip do we want to controller";
 
   JsonObject mode = channelProperties.createNestedObject("mode");
   mode["type"] = "string";
+  mode["description"] = "Select a mode for the strip to be in - color sets a static color, and the strip will change right away, Fade will have the strip change to the designed color within the programmed fade speed set in config, flash will have the strip turn on and off with the rate being controlled via mqtt message";
```

Comment:
I think I have mentioned this - but why do we need "colour" and "fade"? Who doesn't want fade?!

I think this could be simplified to "off", "on", "flash" (although I think flash should be removed also). If flash was removed this could be changed to a simple boolean "state". This becomes much easier to others to figure out how it works.